website/content/v1.1/learn-more/knowledge-base.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Knowledge Base"
+weight: 1999
+description: "Recipes for common configuration tasks with Talos Linux."
+---
+
+## Disabling `GracefulNodeShutdown` on a node
+
+Talos Linux enables [Graceful Node Shutdown](https://kubernetes.io/docs/concepts/architecture/nodes/#graceful-node-shutdown) Kubernetes feature by default.
+
+If this feature should be disabled, modify the `kubelet` part of the machine configuration with:
+
+```yaml
+machine:
+  kubelet:
+    extraArgs:
+      feature-gates: GracefulNodeShutdown=false
```

Comment:
i believe disabling the feature gate is enough

Reply:
it's not - that's the problem, kubelet fails if it finds non-zero settings for graceful node shutdown and the feature gate is disabled, that's what I wasted half of the day on :-D

Reply:
weird :sweat_smile: